lib/rules/rule-empty-line-before/README.md

```diff
@@ -262,6 +262,22 @@ The following patterns are *not* considered violations:
 a {}
 ```
 
+#### `"after-comment"`
```

Comment:
Wrong!

Reply:
Good spot. Would you like to create to PR to fix this?

Reply:
I think I can. But I'm still [not sure that this PR is necessary](https://github.com/stylelint/stylelint/pull/3179#issuecomment-367324023).